tasks/jasmine.js

```diff
@@ -194,7 +194,7 @@ module.exports = function(grunt) {
 
     phantomjs.on('console', function(msg) {
       thisRun.cleanConsole = false;
-      if (options.display === 'full') {
+      if (options.display === 'full' || options.showLogging) {
```

Comment:
Hmm I hadn't considered that someone might want to set `showLogging: false` to hide the logging in "full" mode. I don't particularly object to this...